src/google/protobuf/compiler/objectivec/objectivec_enum.h

```diff
@@ -28,6 +28,36 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+// Protocol Buffers - Google's data interchange format
```

Comment:
It seems like something went wrong here because the copyright notice is duplicated.

Reply:
These files shouldn't be down integrated since their SOT is GitHub. I think we just need to update our file_list.bzl file a little. Will send out a fix for that. Then you will need to close this and run git_sync again.